gem_metrics/bleurt.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+from .metric import ReferencedMetric
+from datasets import load_metric
+import numpy as np
+
+
+class BLEURT(ReferencedMetric):
+    """BLEURT uses the tiny checkpoint for efficient CPU runtime."""
+    def __init__(self, checkpoint_path="bleurt-tiny-128"):
```

Comment:
Is the checkpoint included in the repo?

Reply:
it will automatically download if you don't have it yet.